src/theme/ThemeProvider.js

```diff
@@ -33,6 +33,7 @@ const nwTheme = {
       googleHover: '#fafafa',
       githubHover: '#545159',
     },
+    appDashboardCard: 'rgba(75, 65, 130, 0.2)',
```

Comment:
where is this being used?

Reply:
That's the background color of the card in the application page. Should I name it colors.applicationCard?

Reply:
lmao seems like I forgot to change the hardcoded background-color to use the theme color